src/components/date-time.vue

```diff
@@ -16,22 +16,33 @@ except according to the terms contained in the LICENSE file.
 <script>
 import { DateTime } from 'luxon';
 
-import { formatDate } from '../util/date-time';
+import { formatDateTime } from '../util/date-time';
 
 export default {
   name: 'DateTime',
   props: {
     iso: String, // eslint-disable-line vue/require-default-prop
-    blank: String // eslint-disable-line vue/require-default-prop
+    blank: {
+      type: String,
+      default: ''
+    }
   },
   computed: {
+    fromISO() {
+      return DateTime.fromISO(this.iso);
+    },
+    withLocale() {
+      return this.fromISO.locale === this.$i18n.locale
+        ? this.fromISO
+        : this.fromISO.setLocale(this.$i18n.locale);
+    },
     title() {
-      return this.iso != null
-        ? DateTime.fromISO(this.iso).toFormat('y/MM/dd HH:mm:ss')
-        : null;
+      return this.iso != null ? formatDateTime(this.fromISO) : null;
```

Comment:
Why no locale here?

Reply:
The title shows an absolute time stamp, not a relative date, so no localization takes place. And because we just use `this.fromISO` here instead of `this.withLocale`, if the locale changes, Vue won't have to re-evaluate the title. (I don't think it's a big optimization though, so I think it would have been fine to use `this.withLocale` here.)

Reply:
I see that makes sense. For some reason I thought it could be relative there as well.